CHANGELOG.md

```diff
@@ -2,16 +2,19 @@
 
 ## Master
 
+
+## 2.3.0
+
 ### Bug Fixes
 
 * Parameters in Refract output will no longer contain a `content` value
-  when there is no sample value.  
+  when there is no sample value.
```

Comment:
These double spaces we're done on purpose to trigger a new line in markdown.

![screen shot 2016-04-15 at 16 46 14](https://cloud.githubusercontent.com/assets/44164/14566962/a905984a-0329-11e6-98ea-d2ea7c0f735b.png)

vs

![screen shot 2016-04-15 at 16 46 22](https://cloud.githubusercontent.com/assets/44164/14566967/adc7db72-0329-11e6-88f2-1c1adf5c8336.png)